IPython/html/static/edit/js/main.js

```diff
@@ -31,11 +31,14 @@ require([
 
     var base_url = utils.get_body_data('baseUrl');
     var file_path = utils.get_body_data('filePath');
-    contents = new contents.Contents({base_url: base_url});
     var config = new configmod.ConfigSection('edit', {base_url: base_url});
     config.load();
     var common_config = new configmod.ConfigSection('common', {base_url: base_url});
     common_config.load();
+    contents = new contents.Contents({base_url:base_url, 
+                                        config:config,
```

Comment:
`common_option.config = common_config` may cause problems for the other things we pass common_options to. AIUI, `common_options` is just a convenience thing for 'here are some things we need to pass to several constructors', so if it doesn't make sense to use those things for the Contents constructor, let's just make a separate options dict for it.